### PR TITLE
Update links

### PR DESCRIPTION
**Pull request by jkhall06**
Already done the find and replace on the incorrect links. Test update_links.py with another file/using different links.

- [x] general review code in ons-spark/utilities/update_links.py
- [x] run update_links.py
- [x] spot check updated links in local build to confirm new links work

### CODE REVIEW

**Review by robertswh — COMMENTED** [review]
**General review of the code**
I recommend making the changes commented on specific lines of code. The following comments are more "nice to have". 

Code works for file structure a couple of layers deep. To get this to work for any structure you could put a while loop in to say "while there is a directory still in the list, keep appending files", or I did a quick google to and found os.walk(), e.g. [this post](https://stackoverflow.com/questions/2909975/python-list-directory-subdirectory-and-files).

To develop further you could put this into a class (called, say FindReplace) where each function would be a method from the class. So it could be used something like,
```
docs = FindReplace(old_string, new_string, (".md", ".ipynb))
docs.count_replacements()
docs.find_and_replace()
```
**Running the code**
Code runs fine so the above comments are optional, possibly for future development.

**Check some links**
I found two links didn't work, https://spark.apache.org/docs/latest/api/python/reference/pyspark.sql/api/pyspark.RDD.html
https://spark.apache.org/docs/latest/api/python/_modules/pyspark/sql/utils.html

File: utilities/update_links.py [hunk 1]
```diff
@@ -1,24 +1,34 @@
...
-                       'display.precision', 3)
+                  'display.max_columns', None,
+                  'display.precision', 3)
 
 filepath = r"..\ons-spark\ons-spark"
-old_string = ""
-new_string = ""
+old_string = "sprak"
+new_string = "spark"
 extensions = ('.md', '.ipynb')
```
Comment (robertswh) [inline utilities/update_links.py:10]: lines 7-10 should either be removed or added into the documentation for the wrapper function as an example usage

File: utilities/update_links.py [hunk 2]
```diff
@@ -30,49 +40,78 @@ def _get_files(self):
...
+    docs = FindReplace(filepath, extensions, old_string, new_string)
+    print(docs.count_replacements())
+    docs.find_and_replace()
+    print(docs.count_replacements())
+
+
+
+
+#wrapper function to run
+search_and_replace(filepath, extensions, old_string, new_string)
```
Comment (robertswh) [inline utilities/update_links.py:114]: add this into the documentation for the wrapper function as example usage

**Review by robertswh — COMMENTED** [review]
just a couple of quick things before we can merge

File: utilities/update_links.py (hunk 2 above)
Comment (jkhall06) [reply utilities/update_links.py:114]: added under Example

File: utilities/update_links.py (hunk 1 above)
Comment (jkhall06) [reply utilities/update_links.py:10]: added under Example

**Pull request merged**